flask/requirements.in

```diff
@@ -8,3 +8,4 @@ minio
 pymysql[rsa]
 sqlalchemy
 werkzeug
+pandas
```

Comment:
actually, a very minor thing: could you insert this in its alphabetical position and also ensure that the trailing newline in this file is still there?